src/Mod/Part/App/TopoShapeFacePyImp.cpp

```diff
@@ -647,7 +647,7 @@ PyObject* TopoShapeFacePy::validate(PyObject *args)
             if (!aChecker.IsValid()) {
                 ShapeFix_Shape fix(mkFace.Face());
                 fix.SetPrecision(Precision::Confusion());
-                fix.SetMaxTolerance(Precision::Confusion());
+                fix.SetMinTolerance(Precision::Confusion());

```

Comment:
I am not sure whether it's a good idea to set minimum and maximum tolerance to the same value. It is safer to just eliminate the superfluous call of SetMaxTolerance instead of changing it to SetMinTolerance

Reply:
OK, Werner, you're the boss.
I though it was a copy/paste error (copy/pasting the SetMaxTolerance line and forgetting to change to SetMinTolerance)

Reply:
I guess that with max and min tolerance you define a range but setting them to the same value means that the tolerance must be an exact value and I can imagine that this can lead to problems.
So, removing the line instead at least doesn't change current behaviour.

Reply:
That makes sense to me too.